src/en/howto-juju-block.md

```diff
@@ -0,0 +1,105 @@
+# Restricting changes to the running Juju environment
```

Comment:
Dunno, my tab (Chrome, Firefox) shows this for all pages:

Introducing Charms | Documentation | Juju